tests/setup1/tcase7.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-CONFIG_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+# removed due to SC2034: CONFIG_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
```

Comment:
Did you validate this change? I find it hard to believe that this line had no effect (same for the others)

Reply:
This must has been by accident. Good, that you are reviewing so well what I'm fabricating here 👍🏻 I fixed it and the others as well.